helm/metadata/templates/deployment.yaml

```diff
@@ -158,7 +158,7 @@ spec:
           volumeMounts:
             {{- toYaml . | nindent 10 }}
           {{- end }}
-          {{- with .Values.initResources }}
+          {{- with .Values.resources }}
```

Comment:
Great job with all the helm chart updates here! I see that UC's version is still using initResources in their `master` and `develop` branches:

> https://github.com/uc-cdis/gen3-helm/blob/master/helm/metadata/templates/deployment.yaml#L161C1-L161C44

Is this a divergence that we'll need to maintain?

Reply:
Will change it so that this is a values change not a helm change